setupbase.py

```diff
@@ -17,8 +17,10 @@
 
 from urllib.request import urlopen
 
-import distutils.cmd
-import distutils.log
+try:
+    from setuptools import Command
+except ImportError:
+    from distutils.cmd import Command
```

Comment:
Maybe we should define a requirement for a minimum `setuptools` version to prevent having to do a fallback to `distutils` (and referencing `disutils` in the comments)?

Reply:
This "try-except" is based on setup.py

```python
try:
    from setuptools import setup
    from setuptools.command.install import install
except ImportError:
    from distutils.core import setup
    from distutils.core.command.install import install
```

I will check setuptools minimum version and remove distutils setup.py and setupbase.py.